### DISCUSSION TITLE
支持Netflix的VPS推薦？

### DISCUSSION THREAD

**Discussion by Fubuki0x10DE** [v2fly/v2ray-core#2852] [Q&A]
自己搭建的V2Ray可以打開Prime Video，但打不開Netflix
本人只有支付寶和微信支付
要求支持繁體中文字幕（港區/台區皆可）
本人已經購買了Netflix帳號

**Comment by kingwilliam** [comment]
忘記吧, 可以說netflix已封鎖全球各大中小datacentre IP.

在netflix角度想, 有什麼客戶不是用電訊商IP, 而是用datacentre IP?

利申 : 您想做的已有很多人試了(包括我), 我已試了不同地區的vps 由北美, 南美, 北歐, 東歐, 印度, 日本.

最後 : 我身邊也有成功的方法, 就是買vpn服務(當然 今天可行 不等於明天可行). 還有 我可以 並不等於你可以. 不就是說了等於沒說嗎? 因netflix 會看很多參數, 你用了多久(用了很多年的舊客戶就好大機會成功, 新建立客戶就........), 還有你用什麼付款 如用信用卡又看發卡機構是否等同你開netflix戶口同一地區. 還有很多很多, 所以都是開頭說的 忘記吧!
